docs/conf/portland/2023/news/announcing-cfp.rst

```diff
@@ -19,7 +19,7 @@ Our new venue: Revolution Hall
 Our Portland conference will be held at `Revolution Hall`_. 
 It is a beautiful concert venue located on the `thriving inner east side`_ of Portland.
 It was converted from a high school,
-and still have some lockers in the hallway for that extra learning ambiance.
+and still has some lockers in the hallway for that extra learning ambiance.
```

Comment:
I swear I changed this already. Maybe I changed it in the venue and not in the post. :man_shrugging: